stable/metrics-server/templates/aggregated-metrics-reader-cluster-role.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:aggregated-metrics-reader
```

Comment:
You need to use the fullname prefix for name and same templated labels from the other rbacs.

Reply:
Hmmm, I had thought the name was a special handle, but I can't actually find anything to support that.

I'll try testing with something like:
`name: {{ template "metrics-server.fullname" . }}-aggregated-reader` (and the labels) and re-push if all looks to be working okay.

Reply:
Looks like this works, should address the concerns.